src/definitions/behaviors/api.js

```diff
@@ -157,11 +157,6 @@
                 },
                 write: {
                     cachedResponse: function (url, response) {
-                        if (response && response === '') {
```

Comment:
never true condition, not sure what to do

if response can be a plain text, caching empty response might be expected, as the condition is never true, no BC break

Reply:
Agreed, lets remove this

Reply:
fixed in #2591